lib/govuk_publishing_components/presenters/attachment.rb

```diff
@@ -67,6 +67,10 @@ def unnumbered_reference
         unnumbered_reference
       end
 
+      def is_official_document
+        attachment_data[:command_paper_number].present? || attachment_data[:hoc_paper_number].present?
```

Comment:
We should also show the link if the document is unnumbered, as [per the screenshot on the card](https://trello.com/c/UET4vx6k/1439-update-attachment-component-to-accommodate-new-attachment-design).  A document is still official and can still be ordered (apparently), even if it doesn't have a number (e.g. using the unique ref).

Reply:
Updated, not sure how I missed that.